### DISCUSSION TITLE
Solana: Token is shown "Unknown" in minting process

### DISCUSSION THREAD

**Discussion by moefreelancer83** [phantom/docs#163] [Help & Support] [answered]
### Summary

Hi there,
After setting up our candy machine on Solana the transaction modal for minting does not show what NFT the user is going to receive, which causes the new users don't trust the app. It only shows "+1 Unknown". I assume the phantom looks for the name of NFT and since it is set up with candy machine it cannot get it. Can we make it so that it looks for the collection name and based on that the user can see which collection the NFT belongs to?
Thank you in advance

### Steps to Reproduce

- Setting up a candy machine with Metaplex sugar
- Using UMI as UI  library
- Minting on mainnet
- I use solana wallet adapters
  -    "@solana/wallet-adapter-base": "^0.9.22",
  -     "@solana/wallet-adapter-react": "^0.15.32",
  -     "@solana/wallet-adapter-react-ui": "^0.9.31",
  -     "@solana/wallet-adapter-wallets": "^0.19.16",

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment]
Hi there @moefreelancer83,

Is the metadata set properly? Do you have an example of a token you've minted I can review?

**Reply by moefreelancer83** [reply]
Hi @adamdelphantom ,
Thank you for the reply. Here is the token I minted:
https://solscan.io/token/47SJL3udt5PWtNaaAZRcejtWuhJuaN7vasLEsxEbSycY

I assume the metadata is set correctly if I'm right.

**Reply by moefreelancer83** [reply]
Hi Adam, do you have any update on this please? You think I should mofiy the metadata?

**Comment by stayhungry07212** [comment]
I have the same issue.
It's said Phantom uses the Solana Token Registry to display token information. In order for a token to be displayed in Phantom properly the token project must submit their info to the Solana Token Registry Repo and the Solana team needs to approve it.  But now Solana Token Registry is deprecated and replaced by metaplex standard.
Do Phantom wallet developers try to figure out this issue?

**Reply by adamdelphantom** [reply, marked as answer]
@stayhungry07212 @moefreelancer83 My apologies for the delay in response. [Here](https://docs.phantom.app/best-practices/tokens) is are documentation for displaying tokens in phantom. If your token meets these requirements and it is still not displaying properly, please let me know.
